cvat/apps/engine/backup.py

```diff
@@ -45,7 +44,7 @@
 
 from cvat.apps.engine.location import StorageType, get_location_configuration
 
-class Version(Enum):
+class Version(DjangoEnum, StrEnum):
```

Comment:
I don't think this is needed. This class is not used in models.

Reply:
The reason for these changes is to replace the copy-pasted code for enum methods by a single implementation.

Reply:
But there is no copy-pasted code here.

Reply:
... and in some of the other cases I pointed out, it would be sufficient to inherit from `StrEnum` alone.

Reply:
Ok, I simplified/restored a couple of enums, which don't use additional methods now.